README.md

```diff
@@ -4,6 +4,10 @@ A typesafe and lightweight way to write functions with asynchronous side-effects
 
 [![Build Status](https://travis-ci.org/tp/tsaga.svg?branch=master)](https://travis-ci.org/tp/tsaga)
 
+## Testing
```

Comment:
Should we do anything about the `### Tests` section below? I would  think it's preferable to first show some example usage of the library, then the testing setup.